content/tutorial/tutorial.md

```diff
@@ -14,75 +14,76 @@ redirect_from:
 
 This tutorial doesn't assume any existing React knowledge.
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## ट्यूटोरियल सुरू करण्यापूर्वी {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React app, and mastering it will give you a deep understanding of React.
+या ट्यूटोरियल मध्ये आपण एक छोटासा गेम तयार करू.**आपण कदाचित येथे गेम तयार करण्यासाठी आला नसाल- परंतु एक संधी द्या.** आपण ट्यूटोरियलमध्ये शिकत असलेली टेक्निक कोणतेही अँप तयार करण्यासाठी मूलभूत आहेत आणि त्यामध्ये प्रभुत्व मिळविण्यामुळे आपल्याला प्रिऍक्टचे सखोल ज्ञान मिळेल.
```

Comment:
Please use React only, don't convert it 
प्रिऍक्टचे -> React चे

Reply:
रिऍक्ट or React. What is preferred?
Please comment down below for guiding me!

Reply:
React is preferred. first please check #3

Reply:
Thanks for correcting me. I request you to check the changes. I have followed the above guideline for contribution of translating work. 
Thanks again!